benchexec/tablegenerator/react-table/src/components/SelectColumn.js

```diff
@@ -52,10 +53,39 @@ export default class SelectColumn extends React.Component {
   };
 
   renderColumns = index => {
-    return this.state.list[index].columns.map(column => (
+    this.listTable = [];
+    let idxColumn = [];
+    this.state.list.forEach(tool => {
+      this.selectable.forEach(headerRow => {
+        let length = tool.columns.length;
+        tool.columns.forEach(function(column, idx, i) {
+          if (headerRow.display_title === column.display_title) {
+            idxColumn.push({
+              display_title: column.display_title,
+              isVisible: column.isVisible,
+              key: idx * i.max_width
+            });
+          } else if (
+            idx === length - 1 &&
+            idxColumn
+              .map(elem => elem.display_title.includes(headerRow.display_title))
```

Comment:
`.map().includes()` can be simplified with `.some()`.

Reply:
done